styles/dialog.css

```diff
@@ -66,9 +66,6 @@ ai-dialog > ai-dialog-header > button {
   padding: 0;
   cursor: pointer;
 }
-ai-dialog > ai-dialog-header > button:focus {
```

Comment:
Oh gotcha sorry this was from another PR that hasn't had the finished `dist` and `styles` committed yet to fix an issue with accessibility.  We can ignore the build changes for now until release.